app.py

```diff
@@ -2,37 +2,39 @@
 from urllib.request import urlopen
 import json
 
-# global variables to store repo and repo links
-repos=[]
-repolink=[]
-
-# module to generate repo and repo links
-def repogen(user_name):
-    global repos
-    global repolink
-    # getting contents of the api for a user
-    httpob=urlopen("https://api.github.com/users/"+user_name+"/repos")
-    # decoding the http object recieved
-    decob=httpob.read().decode("utf-8")
-    # converting to json
-    jsonob=json.loads(decob)
-    # appending only personal repos to the list
-    for j in jsonob:
-        if str(j["fork"])=="False":
-            repos.append(j["full_name"])
-            repolink.append("https://www.github.com/"+j["full_name"])
+
+data = {"arindambiswas":{"name":"Arindam Biswas", "gh_handle":"aribis369" ,"fb_link":"https://facebook.com/aribis369", "bio":"to be written by the member", "pic_src":"link"}, "dibyaprakashdas":{"name":"Dibya Prakash Das", "gh_handle":"dibyadas", "fb_link":"https://facebook.com/user_name", "bio":"whatever you like", "pic_src":"link"}}
+
 
 app = Flask(__name__)
 
 
 @app.route('/<name>')
-def main(name):
-    global repos
-    global repolink
-    # generating repo and repolink list before rendering the html file
-    # list to be passed while rendering & some more credentials can be added 
-    repogen(name)
-    return render_template('test.html', name=name)
+def main():
+    mname = data[name]["gh_handle"]
+    repos = []
+    repolink = []
+    blog = str()
+    content={}
+    gh_link = "https://github.com/"+mname
+
+    httpob = urlopen("https://api.github.com/users/"+mname+"/repos")
```

Comment:
Can you change this to use the requests library instead of the urllib module? 
Requests module has many advantages over urllib. It will help us in adding other features in our app in the future. 
It supports a fully restful API.